src/main/js/wiki/components/PageHeader.tsx

```diff
@@ -57,16 +48,19 @@ class PageHeader extends React.Component<Props, State> {
     };
   }
 
-  onCreateClick = () => {
-    this.setState({
-      showCreateForm: true
-    });
+  onOkCreate = (name) => {
```

Comment:
this and any other code belongs to the create page routine of this component is a duplicate from the actioneheader

Reply:
- [X] done